chunkenc/chunk_test.go

```diff
@@ -32,7 +32,7 @@ func TestChunk(t *testing.T) {
 	for enc, nc := range map[Encoding]func() Chunk{
 		EncXOR: func() Chunk { return NewXORChunk() },
 	} {
-		t.Run(fmt.Sprintf("%s", enc), func(t *testing.T) {
+		t.Run(fmt.Sprintf("%d", enc), func(t *testing.T) {
```

Comment:
is there any negative effect if we have this as `t.Run(fmt.Sprintf("%v", enc), func(t *testing.T) {` .
I guess where performance is important it does matter, but in all other cases I prefer to set it as `%v` and forget about it. Less likely to break if the param changes its type.